WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
If you don't understand why there are conditions, you shouldn't modify them in the first place.